src/components/gridList/demoDynamicTiles/index.html

```diff
@@ -1,5 +1,4 @@
 <div ng-app="gridListDemo" ng-controller="gridListDemoCtrl as vm" flex>
-  <md-button ng-click="vm.rotate()">Rotate</md-button>
```

Comment:
Why the removal of the 'rotate' feature? Did you consider it too confusing for visitors?